ompy/error_finder.py

```diff
@@ -0,0 +1,433 @@
+import logging
+import numpy as np
+import pymc3 as pm
+import termtables as tt
+
+from numpy import ndarray
+from typing import Optional, Union, Any, Tuple, List, Dict
+
+
+from .vector import Vector
+from .matrix import Matrix
+from .dist import FermiDirac
+
+
+def all_equal(iterator):
```

Comment:
Suggestion: Move to the end of the file?

Reply:
Good suggestion:)